ota-plus-web/app/reactapp-new/src/components/DocsLinks/DocsLinks.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { useTranslation, UseTranslationResponse } from 'react-i18next';
+import { ExternalLink } from '../common';
+import styled, { DefaultTheme } from 'styled-components';
+
+type Props = {
+  theme: DefaultTheme
+};
+
+const Header = styled.div(({ theme }: Props) => ({
+  color: theme.palette.texts.dark,
+  fontSize: 22,
+  fontWeight: 400,
+  lineHeight: '28px'
+}));
+
+const Description = styled.div(({ theme }: Props) => ({
+  color: theme.palette.texts.black,
+  fontSize: 14,
+  lineHeight: '17px'
+}));
+
+const List = styled.ul((_: Props) => ({
+  listStyle: 'none',
+  margin: 0,
+  padding: 0
+}));
+
+const DocsLinks: React.FC<{}> = () => {
```

Comment:
Don't use React.FC<> to type functional components, it breaks default props and other static properties